src/d3a/models/market/two_sided.py

```diff
@@ -264,9 +280,40 @@ def match_recommendation(self, recommended_list):
                     recommended_list, index+1, trade, bid_trade
                 )
 
+    @staticmethod
+    def validate_requirements_satisfied(
```

Comment:
If this method is used only inside of this class, can we consider making it private?